Index.html

```diff
@@ -101,8 +54,7 @@ <h1>Welcome to Erika Taege's CSD120 Website!</h1>
           I'm usually more reserved when speaking about topic i may not know
           about, but can be very vocal about topics i do know about.
         </p>
-      <div>
-        <table class = "bio">
+        <table>
```

Comment:
The table should not have this class.  It should be inside a div with this class